### ISSUE TITLE
check CKV_SECRET_6 fails even when suppressed

### ISSUE THREAD

**Issue by daniel-intenthq** [bridgecrewio/checkov#1363]
**Describe the bug**
Checkov raises FAILED check CKV_SECRET_6 when running against a project even if the check is suppressed. However if running checkov on a single terraform file it doesn't flag the same error.

**To Reproduce**
Steps to reproduce the behaviour:
1. Create a single terraform file and save in under a new dir (test/main.tf) with a resource would fail this check but make a comment so checkov suppresses the alert:
```
resource "aws_emr_cluster" "cluster" {
  #checkov:skip=CKV_SECRET_6:todo
  #checkov:skip=CKV2_AWS_7:todo
  name          = "emr"
  termination_protection = true
  ebs_root_volume_size = 10
  service_role = "emr_role"
  master_instance_fleet {
    name = "emr-leader"
    target_on_demand_capacity = 1
    instance_type_configs {
      instance_type = var.emr_leader_instance_type

      ebs_config {
        size = "64"
        type = "gp2"
        volumes_per_instance = 1
      }
      configurations {
        classification = "spark-defaults"
        properties = {
          "spark.cassandra.connection.host" = "cassandra.domain.co.uk",
          "spark.cassandra.connection.port" = "9999",
          "spark.cassandra.auth.username" = "user123",
          "spark.cassandra.auth.password" = "pass123", #checkov:skip=CKV_SECRET_6:todo
        }
      }
    }
  }
}
```
2. Run cli command  `checkov -f test/main.tf`
3. Observe result behaves as expected:
```
...
 secrets scan results:

Passed checks: 0, Failed checks: 0, Skipped checks: 1

Check: CKV_SECRET_6: "Base64 High Entropy String"
	SKIPPED for resource: d34df506cfa43c623b4da8853ce91637ed575be6
	Suppress comment: checkov:skip=
	File: /test/main.tf:29-30
	Guide: https://docs.bridgecrew.io/docs/git_secrets_6
```
4. Run cli command `checkov --directory test/`
5. Observe the same test fail:
```
...
secrets scan results:

Passed checks: 0, Failed checks: 1, Skipped checks: 0

Check: CKV_SECRET_6: "Base64 High Entropy String"
	FAILED for resource: d34df506cfa43c623b4da8853ce91637ed575be6
	File: /main.tf:29-30
	Guide: https://docs.bridgecrew.io/docs/git_secrets_6

		29 |
```

**Expected behaviour**
There is no terraform code difference, checkov should behave the same way in both tests, it should show SKIPPED for CKV_SECRET_6.

**Desktop (please complete the following information):**
 - Terraform 1.0.1
 - Checkov Version 2.0.259

**Additional context**
I have not been able to observe if this behaviour is unique to check CKV_SECRET_6 or not. But it's the first time I have observed a suppressed check still show as failure.

**Comment by gruebel** [comment]
super interesting, it works when you are in the folder and do the check like
```shell
checkov -d .
```
but it fails, when doing it from the parent folder, like you did it.
